test/functional/test-resources.js

```diff
@@ -1173,20 +1144,19 @@ describe('Resources changeSize', () => {
     it('should force resize after focus', () => {
       resources.scheduleChangeSize_(resource1, 111, 222, false);
       resources.mutateWork_();
-      expect(resource1.getPendingChangeSize().height).to.equal(111);
-      expect(resource1.getPendingChangeSize().width).to.equal(222);
-      expect(resources.requestsChangeSize_.length).to.equal(0);
+      expect(resource1.getPendingChangeSize()).to.jsonEqual(
+          {height: 111, width: 222});
+      expect(resources.requestsChangeSize_).to.be.empty;
 
       resources.checkPendingChangeSize_(resource1.element);
       expect(resource1.getPendingChangeSize()).to.be.undefined;
       expect(resources.requestsChangeSize_.length).to.equal(1);
 
       resources.mutateWork_();
-      expect(resources.requestsChangeSize_.length).to.equal(0);
-      expect(resource1.changeSize.callCount).to.equal(1);
-      expect(resource1.changeSize.firstCall.args[0]).to.equal(111);
-      expect(resource1.changeSize.firstCall.args[1]).to.equal(222);
-      expect(overflowCallbackSpy.callCount).to.equal(2);
+      expect(resources.requestsChangeSize_).to.be.empty;
+      expect(resource1.changeSize).to.be.calledOnce;
+      expect(resource1.changeSize.firstCall.args).to.jsonEqual([111, 222]);
```

Comment:
I'm not sure about this, but could you try if chai-sinon support syntax like `to.be.calledWith()`, or something similar?